diff --git a/node_modules/@hydrooj/framework/validator.ts b/node_modules/@hydrooj/framework/validator.ts
index 0766483..30eab33 100644
--- a/node_modules/@hydrooj/framework/validator.ts
+++ b/node_modules/@hydrooj/framework/validator.ts
@@ -7,7 +7,7 @@ import Schema from 'schemastery';
 type InputType = string | number | Record<string, any> | any[];
 export type Converter<T> = (value: any) => T;
 export type Validator<Loose extends boolean = true> = (value: Loose extends true ? any : InputType) => boolean;
-export type Type<T> = Schema<T> | readonly [Converter<T>, Validator<false>?, (boolean | 'convert')?];
+export type Type<T> = Schema<T> | readonly [Converter<T>, (Validator<false> | null)?, (boolean | 'convert')?];
 
 const MaybeArray = <T>(inner: Schema<T>) => Schema.union([Schema.array(inner), inner]);
 type CheckFunction = <IsNumber extends boolean>(v: IsNumber extends true ? number : string) => boolean;
@@ -89,7 +89,7 @@ const saslprepString = <T = string>(regex?: RegExp, cb?: (i: string) => boolean,
 ] as [(v) => string, (v) => boolean];
 
 export const Types = {
-    Content: [(v) => v.toString().trim(), (v) => v?.toString()?.trim() && v.toString().trim().length < 65536],
+    Content: [(v) => v.toString().trim(), (v) => !!v?.toString()?.trim() && v.toString().trim().length < 65536],
     Key: saslprepString(/^[\w-]{1,255}$/),
     /** @deprecated */
     Name: saslprepString(/^.{1,255}$/),
